recipes/openssl/1.x.x/test_package/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(test_package C)
 option(OPENSSL_WITH_ZLIB "OpenSSL with zlib support" ON)
 
 set(OpenSSL_DEBUG 1)
-find_package(OpenSSL REQUIRED)
+find_package(OpenSSL REQUIRED CONFIG)
```

Comment:
We should be testing the Official module works above the config version.

The docs I havent written and linked

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/test_packages.md#testing-cmake-variables-from-findmodules

Reply:
I agree, please keep `find_package(OpenSSL REQUIRED)` and `cmake_find_package` generator in test v1 package.

Reply:
Done, reverted to the regular FindOpenSSL.cmake. Thank you for checking it!